### PR TITLE
Add failing test fixture for IssetOnPropertyObjectToPropertyExistsRector

### PR DESCRIPTION
**Pull request by vladyslavstartsev**
# Failing Test for IssetOnPropertyObjectToPropertyExistsRector

Based on https://getrector.org/demo/051dc5d8-cb1e-4f72-b916-cf5dae1ee4d2

If it's typed property AND it's strict (not nullable), it's ok for it to check `isset` instead of null

### CODE REVIEW

**Comment by TomasVotruba** [comment]
The fixture name DemoFile is not very descriptive. Should the fixture be skipped or modified? 
We have prefix `Skip*` naming convention for skipped files, so they're easy to spot in IDE.

Could you update it, so it's more clear? Thank you

**Comment by vladyslavstartsev** [comment]
@TomasVotruba done

**Comment by TomasVotruba** [comment]
Thanks. I'm not sure what `AbstractAdmin` means. It should be something like e.g. `skip_class_with_parent.php.inc`, so the intention is obvious from the fixture name.

File: rules/code-quality/tests/Rector/Isset_/IssetOnPropertyObjectToPropertyExistsRector/Fixture/AbstractAdmin.php.inc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            return $this->cachedBaseRoutePattern;
+        }
+
+        $this->cachedBaseRoutePattern = sprintf(
+            '/%s%s',
+            $matches[1] === '' ? '' : $this->urlize($matches[1], '-') . '/',
+            $this->urlize($matches[3], '-')
+        );
+
+        return $this->cachedBaseRoutePattern;
```
Comment (TomasVotruba) [inline rules/code-quality/tests/Rector/Isset_/IssetOnPropertyObjectToPropertyExistsRector/Fixture/AbstractAdmin.php.inc:21]: How is this part relevant to the use case? Is it require to make it fail?

**Comment by vladyslavstartsev** [comment]
`AbstractAdmin` it was actually class of Sonata Admin Bundle

File: rules/code-quality/tests/Rector/Isset_/IssetOnPropertyObjectToPropertyExistsRector/Fixture/skip_isset_change_if_typed_property.php.inc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    private string $cachedBaseRoutePattern;
+    
+    public function getBaseRoutePattern(): string
+    {
+        if (isset($this->cachedBaseRoutePattern)) {
+            return $this->cachedBaseRoutePattern;
+        }
+    }
+}
+?>
```
Comment (TomasVotruba) [inline rules/code-quality/tests/Rector/Isset_/IssetOnPropertyObjectToPropertyExistsRector/Fixture/skip_isset_change_if_typed_property.php.inc:16]: Purfect, thanks you :clap:

**Comment by TomasVotruba** [comment]
Here is failing example, if the property is compare to null:
https://3v4l.org/fLrEg
